owncloudApp/src/main/java/com/owncloud/android/ui/preview/PreviewTextFragment.java

```diff
@@ -329,55 +335,20 @@ public void onPrepareOptionsMenu(Menu menu) {
         super.onPrepareOptionsMenu(menu);
 
         if (mContainerActivity.getStorageManager() != null) {
-            FileMenuFilter mf = new FileMenuFilter(
-                    getFile(),
-                    mContainerActivity.getStorageManager().getAccount(),
-                    mContainerActivity,
-                    getActivity()
-            );
-            mf.filter(
-                    menu,
-                    false,
-                    false,
-                    false,
-                    false
+            OCFile safeFile = getFile();
```

Comment:
This whole code seems to need refactoring. I assume that this is beyond the scope of this issue.

Reply:
Exactly, 2 out of the 4 preview fragments are still in Java and need to be refactored, and even the other 2 need to be moved to the new architecture as well, so we'll create a issue for that out of the scope of this one 👍